src/dirs.rs

```diff
@@ -1,21 +1,71 @@
+#[cfg(not(target_os="android"))]
 use appdirs;
 
+use std::fs;
 use std::path::PathBuf;
 
+#[cfg(target_os="android")]
+static mut ANDROID_DATA_DIR: Option<PathBuf> = None;
+
 pub fn get_config_dir() -> Result<PathBuf, ()> {
-    let mut dir = appdirs::user_config_dir(Some("activitywatch"), None, false)?;
-    dir.push("aw-server-rust");
-    return Ok(dir);
+    #[cfg(not(target_os="android"))]
+    {
+        let mut dir;
+        dir = appdirs::user_config_dir(Some("activitywatch"), None, false)?;
+        dir.push("aw-server-rust");
+        return Ok(dir);
+    }
+
+    #[cfg(target_os="android")]
+    {
+        return Err(());
```

Comment:
@johan-bjareholt Maybe I should return proper error messages?

Reply:
I think a panic! would be fine here. It would be a programmer error to try and use this on android when it's not implemented for that platform.

Reply:
Ah, makes sense, will fix that.